scripts/rep-faucet.js

```diff
@@ -15,7 +15,7 @@ function faucetInAndMigrate(augur, universe, auth, callback) {
     augur.api.ReputationToken.balanceOf({ _owner: auth.address, tx: { to: reputationToken } }, function (err, reputationTokenBalance) {
       if (err) return callback(err);
       console.log("reputationTokenBalance:", reputationTokenBalance);
-      if (new BigNumber(reputationTokenBalance, 10).gt(new BigNumber("100000", 10))) return callback(null);
```

Comment:
Might be more correct to make this new `Error("Already loaded")` but not too big a deal for a one-off script

Reply:
I'll make the change